api/controllers/game/stalemate-reject.js

```diff
@@ -0,0 +1,30 @@
+module.exports = async function (req, res) {
+  try {
+    const game = await Game.findOne({ id: req.session.game }).populate('players');
+    let gameUpdates = {
+      turnStalemateWasRequestedByP0: -1,
+      turnStalemateWasRequestedByP1: -1,
```

Comment:
I feel like these should be `null` instead since it's a meaningful choice. Though this way you get to always know it's a number which is also fine with me.

Reply:
Yeah this was mostly b/c of database adapter type safety. Basically sails gets pissy using null for number columns despite the fact that it's actually fine in postgres